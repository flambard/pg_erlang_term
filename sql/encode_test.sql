select erlang_term_encode('[]');
select erlang_term_encode('erlang');
