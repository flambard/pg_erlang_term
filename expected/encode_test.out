select erlang_term_encode('[]');
 erlang_term_encode 
--------------------
 \x83836a
(1 row)

select erlang_term_encode('erlang');
   erlang_term_encode   
------------------------
 \x8383770665726c616e67
(1 row)

